server/auth/types/openid/openid_auth.ts

```diff
@@ -296,8 +296,9 @@ export class OpenIdAuthentication extends AuthenticationType {
           cookie.credentials = {
             authHeaderValueExtra: true,
             refresh_token: refreshTokenResponse.refreshToken,
+            expiryTime: getExpirationDate(refreshTokenResponse),
           };
-          cookie.expiryTime = getExpirationDate(refreshTokenResponse);
+          cookie.expiryTime = Date.now() + this.config.session.ttl;
```

Comment:
If there is a session `ttl` that is longer than the original `expiryTime`, will this be also added into the updated `expiryTime`?

Reply:
session `ttl` is only for cookie, if `expiryTime` is less than session `ttl` and refresh token is also invalidated or not present then `isCookieValid` function will return false and cookie will get invalidated.